icx/wallet/wallet.py

```diff
@@ -220,4 +221,35 @@ def get_address(self):
         """
         return self.address
 
+    @staticmethod
```

Comment:
Use @classmethod, not @staticmethod. See [this](https://github.com/shoark7/Effective-Python/blob/master/files/BetterWay24_classmethod.md) document.

Reply:
Fixed it.